code/modules/mob/living/say.dm

```diff
@@ -321,12 +321,12 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(can_speak_basic(message) && can_speak_vocal(message))
 		return 1
 
-/mob/living/proc/can_speak_basic(message, ignore_spam = FALSE) //Check BEFORE handling of xeno and ling channels
+/mob/living/proc/can_speak_basic(message, ignore_spam = FALSE, forced = null) //Check BEFORE handling of xeno and ling channels
```

Comment:
```suggestion
/mob/living/proc/can_speak_basic(message, ignore_spam = FALSE, forced = FALSE) //Check BEFORE handling of xeno and ling channels
```

Reply:
Not sure if works.

`/mob/living/say(message, bubble_type,var/list/spans = list(), sanitize = TRUE, datum/language/language = null, ignore_spam = FALSE, forced = null)
...
can_speak_basic(original_message, ignore_spam, forced)`

Where "forced" is string. Not sure how conversions work in bcode.